### DISCUSSION TITLE
Peer dependency Issue running Yarn in StableStudio directory?

### DISCUSSION THREAD

**Discussion by Myco-Heterotroph** [Stability-AI/StableStudio#48] [Q&A]
Hello, hoping to get some assistance I appear to be missing something and Im not sure what to try next.
I checked my versions, and this is what I have.
node --version v12.22.9
npm --version 8.5.1
yarn --version 1.22.19

When I am in the StableStudio directory and I run Yarn I get the following issues.  Any idea at what I need to address?

> ➤ YN0000: ┌ Resolution step
> ➤ YN0002: │ @stability/stablestudio-plugin-example@workspace:packages/stablestudio-plugin-example doesn't provide @types/node (pe4c4e), requested by ts-node
> ➤ YN0002: │ @stability/stablestudio-plugin-webgpu@workspace:packages/stablestudio-plugin-webgpu doesn't provide @types/node (p1e18d), requested by ts-node
> ➤ YN0002: │ @stability/stablestudio-plugin-webui@workspace:packages/stablestudio-plugin-webui doesn't provide @types/node (p09690), requested by ts-node
> ➤ YN0002: │ @stability/stablestudio-plugin@workspace:packages/stablestudio-plugin doesn't provide @types/node (pb7ed9), requested by ts-node
> ➤ YN0002: │ notistack@npm:3.0.1 [47cbf] doesn't provide csstype (p2d22e), requested by goober
> ➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
> ➤ YN0000: └ Completed
> ➤ YN0000: ┌ Fetch step
> ➤ YN0000: └ Completed
> ➤ YN0000: ┌ Link step
> ➤ YN0000: └ Completed
> ➤ YN0000: Done with warnings in 0s 552ms

**Comment by charkour** [comment]
I was able to run it with node v18. You could try updating versions because Node 12 has not been support for awhile

**Comment by Myco-Heterotroph** [comment]
Oops I miss-typed on that one line.   I have Node.js v20.1.0
But .....with your suggestion I decided to downgrade to node v18 
That worked! Thank you!
